source/array-index.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
+Extract the keys of an array or tuple.
```

Comment:
The description here should be the same as in the readme.

I also think "keys" may be slightly confusing.